src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(x, y) {
+      return x + y;
+    },
+    subtract(x, y) {
+      return x - y;
+    },
+    multiply(x, y) {
```

Comment:
```suggestion
    result: 0,
    
    add(x, y) {
      return x + y;
    },
    
    subtract(x, y) {
      return x - y;
    },
    
    multiply(x, y) {
```
Для кращої читабельності додавай пусті рядки